tests/test_connection.py

```diff
@@ -53,7 +53,7 @@ def test_connect(node_factory):
         l1.rpc.connect('032cf15d1ad9c4a08d26eab1918f732d8ef8fdc6abb9640bf3db174372c491304e')
 
     # Should get reasonable error if connection refuse.
-    with pytest.raises(RpcError, match=r'Connection establishment: Connection refused'):
+    with pytest.raises(RpcError, match=r'Connection establishment: (Connection refused|Bad file descriptor)'):
```

Comment:
In this case, I usualy preferer to add a comment that points out that we can receive different err message for different os. However, maybe we could be interested only in the prefix of the error?